Version 9.0.0

JohnKendrick opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

JohnKendrick committed:
Update meta.yaml

Updated the sha256 key

JohnKendrick committed:
Merge pull request #3 from JohnKendrick/JohnKendrick-patch-1

Version 9.0.0

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ top-level output has some malformed specs:
- In section run: `blas =*=openblas`
Requirement spec fields should match the syntax `name [version [build]]`to avoid known issues in conda-build. For example, instead of `name =version=build`, use `name version.* build`. There should be no spaces between version operators and versions either: `python >= 3.8` should be `python >=3.8`.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/15942061538](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15942061538). Examine the logs at this URL for more detail.</sub>

JohnKendrick committed:
Added spglib

Added spglib and a new format for the minimum python

JohnKendrick committed:
Removed license_family

Added a link to phonopy-pdielec-born

JohnKendrick committed:
Version 9.0.0 Release

Changed the version number
Changed the way the minimum python is specified
Added phonopy-pdielec-born command
Change the way the license file was give

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ license_file entry is missing, but is required.

For **recipe/meta.yaml**:

 * ℹ️ top-level output has some malformed specs:
- In section run: `blas =*=openblas`
Requirement spec fields should match the syntax `name [version [build]]`to avoid known issues in conda-build. For example, instead of `name =version=build`, use `name version.* build`. There should be no spaces between version operators and versions either: `python >= 3.8` should be `python >=3.8`.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/15946946704](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15946946704). Examine the logs at this URL for more detail.</sub>

JohnKendrick committed:
Put the license_file: back

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ top-level output has some malformed specs:
- In section run: `blas =*=openblas`
Requirement spec fields should match the syntax `name [version [build]]`to avoid known issues in conda-build. For example, instead of `name =version=build`, use `name version.* build`. There should be no spaces between version operators and versions either: `python >= 3.8` should be `python >=3.8`.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/15947096582](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15947096582). Examine the logs at this URL for more detail.</sub>

JohnKendrick committed:
Removed license:

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ The license item is expected in the about section.

For **recipe/meta.yaml**:

 * ℹ️ top-level output has some malformed specs:
- In section run: `blas =*=openblas`
Requirement spec fields should match the syntax `name [version [build]]`to avoid known issues in conda-build. For example, instead of `name =version=build`, use `name version.* build`. There should be no spaces between version operators and versions either: `python >= 3.8` should be `python >=3.8`.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/15947124856](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15947124856). Examine the logs at this URL for more detail.</sub>
